### ISSUE TITLE
Is there way to stop/start/restart the app

### ISSUE THREAD

**Issue by dpvenkatesh** [githubsaturn/captainduckduck#48]
Is there way to stop/start/restart the app from Captain or CLI?

**Comment by githubsaturn** [comment]
Yes. Simply clicking on "Save Configuration & Update" on Captain dashboard will restart the app.

**Comment by dpvenkatesh** [comment]
Can we stop the app

**Comment by githubsaturn** [comment]
Yes. Simply set the instance count to 0.

**Comment by dpvenkatesh** [comment]
Tried setting the instance count to 0, app was still accessible. 
Later I tried to set instance count to 10. Captain and APP went down. I restarted the DROPLET and ran `un -v /var/run/docker.sock:/var/run/docker.sock dockersaturn/captainduckduck` in my droplet
got below error, 

Captain Starting ...
Installing Captain Service ...
December 17th 2017, 8:30:48.462 pm    Starting swarm at 139.59.93.238:2377
Installation failed.
{ Error: (HTTP code 503) unexpected - This node is already part of a swarm. Use "docker swarm leave" to leave this swarm and join another one.
    at /usr/src/app/node_modules/docker-modem/lib/modem.js:254:17
    at getCause (/usr/src/app/node_modules/docker-modem/lib/modem.js:284:7)
    at Modem.buildPayload (/usr/src/app/node_modules/docker-modem/lib/modem.js:253:5)
    at IncomingMessage.<anonymous> (/usr/src/app/node_modules/docker-modem/lib/modem.js:229:14)
    at emitNone (events.js:110:20)
    at IncomingMessage.emit (events.js:207:7)
    at endReadableNT (_stream_readable.js:1057:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
  reason: undefined,
  statusCode: 503,
  json:
   { message: 'This node is already part of a swarm. Use "docker swarm leave" to leave this swarm and join another one.' } }
npm info lifecycle captain-duckduck@0.0.0~poststart: captain-duckduck@0.0.0
npm info ok



captain and app both are not coming up.
